packages/cookbook/transactions/meta-transaction-relayer.js

```diff
@@ -0,0 +1,50 @@
+const { Account } = require('@near-js/accounts');
+const { UnencryptedFileSystemKeyStore } = require('@near-js/keystores-node');
+const { JsonRpcProvider } = require('@near-js/providers');
+const { InMemorySigner } = require('@near-js/signers');
+const { actionCreators } = require('@near-js/transactions');
+const BN = require('bn.js');
+const os = require('os');
+const path = require('path');
+
+const { transfer } = actionCreators;
+
+async function sendNearThroughRelayer({ amount, receiverId, senderAccount }) {
+    const response = await senderAccount.signAndSendSignedDelegate({
+        actions: [transfer(amount)],
+        receiverId,
+        relayerUrl: 'https://relayer.org/relay',
+    });
+
+    console.log(response);
+    return response;
+}
+
+module.exports = {
+    sendNearThroughRelayer,
```

Comment:
I created the `cookbook` examples from a new template from an in-review PR that supports testing snippets here. When that PR is landed I can add integration tests for these.